src/emitter.ts

```diff
@@ -0,0 +1,16 @@
+type Subscriber<T extends any[]> = (...args: T) => void;
+
+export class Emitter<T extends any[]> {
```

Comment:
lets remove this since it's not needed now?

Reply:
yep